<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DreamRadar</class>
 <widget class="QWidget" name="DreamRadar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dream Radar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_26">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileMACAddress">
        <property name="text">
         <string>MAC Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileMACAddressValue"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileDSType">
        <property name="text">
         <string>DS Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileDSTypeValue"/>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileVCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileVCountValue"/>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileTimer0">
        <property name="text">
         <string>Timer0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileTimer0Value"/>
      </item>
      <item row="0" column="10" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelProfileGxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="labelProfileGxStatValue"/>
      </item>
      <item row="1" column="11">
       <widget class="QLabel" name="labelProfileVFrame">
        <property name="text">
         <string>VFrame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="labelProfileVFrameValue"/>
      </item>
      <item row="0" column="13" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QLabel" name="labelProfileKeypresses">
        <property name="text">
         <string>Keypresses</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="labelProfileKeypressesValue"/>
      </item>
      <item row="1" column="14">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QLabel" name="labelProfileGameValue"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="TabWidget" name="tabRNGSelector">
     <widget class="QWidget" name="tabGenerator">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">100</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSlot1">
            <property name="text">
             <string>Slot 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie1"/>
          </item>
          <item row="0" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorSlot2">
            <property name="text">
             <string>Slot 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie2"/>
          </item>
          <item row="1" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorSlot3">
            <property name="text">
             <string>Slot 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie3"/>
          </item>
          <item row="2" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender3"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorSlot4">
            <property name="text">
             <string>Slot 4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie4"/>
          </item>
          <item row="3" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender4"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelGeneratorSlot5">
            <property name="text">
             <string>Slot 5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie5"/>
          </item>
          <item row="4" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender5"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelGeneratorSlot6">
            <property name="text">
             <string>Slot 6</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorSpecie6"/>
          </item>
          <item row="5" column="2">
           <widget class="ComboBox" name="comboBoxGeneratorGender6"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelGeneratorBadges">
            <property name="text">
             <string>Badges</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBoxGeneratorBadges">
            <property name="maximum">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterGenerator" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewGenerator"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxSearcherInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxSearcherMaxAdvances">
            <property name="text">
             <string notr="true">100</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherStartDate">
            <property name="text">
             <string>Start Date</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="DateEdit" name="dateEditSearcherStartDate"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSearcherEndDate">
            <property name="text">
             <string>End Date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="DateEdit" name="dateEditSearcherEndDate"/>
          </item>
          <item row="4" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxSearcherSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherSlot1">
            <property name="text">
             <string>Slot 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie1"/>
          </item>
          <item row="0" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender1"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherSlot2">
            <property name="text">
             <string>Slot 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie2"/>
          </item>
          <item row="1" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherSlot3">
            <property name="text">
             <string>Slot 3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie3"/>
          </item>
          <item row="2" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender3"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSearcherSlot4">
            <property name="text">
             <string>Slot 4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie4"/>
          </item>
          <item row="3" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender4"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelSearcherSlot5">
            <property name="text">
             <string>Slot 5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie5"/>
          </item>
          <item row="4" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender5"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelSearcherSlot6">
            <property name="text">
             <string>Slot 6</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="ComboBox" name="comboBoxSearcherSpecie6"/>
          </item>
          <item row="5" column="2">
           <widget class="ComboBox" name="comboBoxSearcherGender6"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelSearcherBadges">
            <property name="text">
             <string>Badges</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBoxSearcherBadges">
            <property name="maximum">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxSearcherFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_24">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterSearcher" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBar"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="TableView" name="tableViewSearcher">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>Form/Controls/TabWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Form/Controls/DateEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Form/Controls/DateEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabRNGSelector</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxGeneratorSpecie1</tabstop>
  <tabstop>comboBoxGeneratorGender1</tabstop>
  <tabstop>comboBoxGeneratorSpecie2</tabstop>
  <tabstop>comboBoxGeneratorGender2</tabstop>
  <tabstop>comboBoxGeneratorSpecie3</tabstop>
  <tabstop>comboBoxGeneratorGender3</tabstop>
  <tabstop>comboBoxGeneratorSpecie4</tabstop>
  <tabstop>comboBoxGeneratorGender4</tabstop>
  <tabstop>comboBoxGeneratorSpecie5</tabstop>
  <tabstop>comboBoxGeneratorGender5</tabstop>
  <tabstop>comboBoxGeneratorSpecie6</tabstop>
  <tabstop>comboBoxGeneratorGender6</tabstop>
  <tabstop>spinBoxGeneratorBadges</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>textBoxSearcherInitialAdvances</tabstop>
  <tabstop>textBoxSearcherMaxAdvances</tabstop>
  <tabstop>dateEditSearcherStartDate</tabstop>
  <tabstop>dateEditSearcherEndDate</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxSearcherSpecie1</tabstop>
  <tabstop>comboBoxSearcherGender1</tabstop>
  <tabstop>comboBoxSearcherSpecie2</tabstop>
  <tabstop>comboBoxSearcherGender2</tabstop>
  <tabstop>comboBoxSearcherSpecie3</tabstop>
  <tabstop>comboBoxSearcherGender3</tabstop>
  <tabstop>comboBoxSearcherSpecie4</tabstop>
  <tabstop>comboBoxSearcherGender4</tabstop>
  <tabstop>comboBoxSearcherSpecie5</tabstop>
  <tabstop>comboBoxSearcherGender5</tabstop>
  <tabstop>comboBoxSearcherSpecie6</tabstop>
  <tabstop>comboBoxSearcherGender6</tabstop>
  <tabstop>spinBoxSearcherBadges</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
